cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -111,6 +117,23 @@ export default {
       fileButton.has({ disabled: true }),
     ]);
   },
+  verifySetCriteriaPaneElements() {
```

Comment:
this is a redundant method, please use similar existing ones

Reply:
I didn't find can directly replace the function, if you think you can use “verifySetCriteriaPaneItems” to replace, the need for additional permissions "permissions. BulkEditLogsView. GUI", Also, I don't see a function that checks only the following three "not selected" radio buttons
'Users'
'Inventory - items'
'Inventory - holdings'

That's why I started all over again